src/App.tsx

```diff
@@ -1,25 +1,116 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [visibilityError, setVisibilityTheError] = useState(true);
+  const [userId, setUserId] = useState(0);
+  const [userChosen, setUserChosen] = useState(true);
```

Comment:
Same here
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig